inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -26,22 +26,22 @@ class SourceParameters:
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationAddSourceParameters:
-    file_name: str
-    sources: list[str] = field(default_factory=lambda: [])
+    source_list_file_name: str
```

Comment:
The renaming of `file_name` to `source_list_file_name` is clearer, but ensure that all references to this field throughout the codebase are updated accordingly. [important]